src/frontend/mame/ui/slotopt.cpp

```diff
@@ -28,46 +30,9 @@ namespace {
 void *ITEMREF_RESET = ((void *)1);
 char DIVIDER[] = "------";
 
-
-//-------------------------------------------------
-//  get_slot_length
-//-------------------------------------------------
-
-int get_slot_length(const device_slot_interface &slot)
-{
-	int val = 0;
-	for (auto &option : slot.option_list())
-		if (option.second->selectable())
-			val++;
-
-	return val;
-}
-
-
-//-------------------------------------------------
-//  get_slot_option
-//-------------------------------------------------
-
-const char *get_slot_option(device_slot_interface &slot, int index)
-{
-	if (index >= 0)
-	{
-		int val = 0;
-		for (auto &option : slot.option_list())
-		{
-			if (val == index)
-				return option.second->name();
-
-			if (option.second->selectable())
-				val++;
-		}
-	}
-
-	return "";
-}
+};
```

Comment:
Please don't place superfluous semicolons after the closing brace of a namespace.